.github/workflows/authors.yml

```diff
@@ -0,0 +1,23 @@
+name: "authors update"
+on:
+  schedule:
+    # 00:00:00 on the first day of every month
+    # https://crontab.guru/#0_0_1_1-12_*
+    - cron: "0 0 * 1-12 *"
+  workflow_dispatch:
```

Comment:
What's this for?

Reply:
it allows maintainers to trigger the action manually. You'll get a button like this

<img width="975" alt="image" src="https://user-images.githubusercontent.com/39992/97344760-708d0700-1846-11eb-8ede-6b30566905b3.png">

(without the "Version of https://www.npmjs.com/package/@github/openapi" input). See https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#manual-events

Reply:
It adds a manual trigger that looks like [this](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/).

Reply:
Ah, that's certainly useful, thanks.